chain/client-primitives/Cargo.toml

```diff
@@ -13,7 +13,6 @@ description = "This crate hosts NEAR client-related error types"
 [dependencies]
 actix = "=0.11.0-beta.2"
 chrono = { version = "0.4.4", features = ["serde"] }
-serde = { version = "1", features = ["derive"] }
```

Comment:
As alternative, we can use a negative import here: 

```toml
# Note: this crate intentionally doesn't depend on serde
# serde = "1"
```

I think this might be moderately valuable while we are in-transition, and most things do depend on serde.

Reply:
It's a good idea. However, I'm not sure we need it.